### ISSUE TITLE
Logs are not re-loaded after changing logging level

### ISSUE THREAD

**Issue by meichthys** [nextcloud/logreader#754]
### Steps to reproduce
1. Open Settings
2. Navigate to Logging
3. Change Logging level to include debug/info

### Expected behaviour
The logs should be re-loaded with the new logging level included

### Actual behaviour
The logging page is blank 

### Server configuration

**Nextcloud version:** (see Nextcloud admin page)
24
**Where did you install Nextcloud from:**
Nextcloud VM

**List of activated apps:**
```
Enabled:
  - accessibility: 1.10.0
  - admin_audit: 1.14.0
  - bruteforcesettings: 2.4.0
  - calendar: 3.4.2
  - cloud_federation_api: 1.7.0
  - comments: 1.14.0
  - contacts: 4.1.1
  - contactsinteraction: 1.5.0
  - cookbook: 0.9.13
  - dav: 1.22.0
  - deck: 1.7.1
  - drawio: 1.0.3
  - external: 4.0.0
  - extract: 1.3.5
  - federatedfilesharing: 1.14.0
  - files: 1.19.0
  - files_antivirus: 3.3.0
  - files_automatedtagging: 1.14.0
  - files_fulltextsearch: 24.0.0
  - files_fulltextsearch_tesseract: 24.0.0
  - files_pdfviewer: 2.5.0
  - files_retention: 1.13.1
  - files_rightclick: 1.3.0
  - files_sharing: 1.16.2
  - files_trashbin: 1.14.0
  - files_versions: 1.17.0
  - files_videoplayer: 1.13.0
  - files_zip: 1.1.2
  - firstrunwizard: 2.13.0
  - forms: 2.5.1
  - fulltextsearch: 24.0.0
  - fulltextsearch_elasticsearch: 24.0.0
  - guests: 2.2.0
  - impersonate: 1.11.0
  - logreader: 2.9.0
  - lookup_server_connector: 1.12.0
  - maps: 0.1.10
  - metadata: 0.16.0
  - nextcloud_announcements: 1.13.0
  - notifications: 2.12.0
  - oauth2: 1.12.0
  - occweb: 0.1.0
  - password_policy: 1.14.0
  - photos: 1.6.0
  - previewgenerator: 5.0.0
  - privacy: 1.8.0
  - provisioning_api: 1.14.0
  - rainloop: 7.2.5
  - recommendations: 1.3.0
  - richdocuments: 6.1.1
  - serverinfo: 1.14.0
  - settings: 1.6.0
  - sharebymail: 1.14.0
  - side_menu: 2.4.1
  - socialsharing_email: 2.5.0
  - systemtags: 1.14.0
  - tasks: 0.14.4
  - text: 3.5.1
  - theming: 1.15.0
  - twofactor_backupcodes: 1.13.0
  - twofactor_totp: 6.4.0
  - updatenotification: 1.14.0
  - user_migration: 1.0.4
  - user_status: 1.4.0
  - video_converter: 1.0.5
  - viewer: 1.8.0
  - weather_status: 1.4.0
  - workflow_ocr: 1.24.3
  - workflowengine: 2.6.0
Disabled:
  - activity: 2.16.0
  - circles: 23.0.1
  - dashboard: 7.4.0
  - encryption
  - facerecognition
  - federation: 1.13.0
  - files_external: 1.15.0
  - support: 1.6.0
  - survey_client: 1.11.0
  - user_ldap
```

### Client configuration
**Browser:**
Firefox (also present in Edge)

#### Browser log
```
Error: Minified React error #188; visit https://reactjs.org/docs/error-decoder.html?invariant=188 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    findDOMNode React
    transition CSSTransitionGroupChild.js:129
    componentWillLeave CSSTransitionGroupChild.js:103
    performLeave TransitionGroup.js:108
    componentDidUpdate TransitionGroup.js:198
    componentDidUpdate TransitionGroup.js:197
    React 2
    unstable_runWithPriority scheduler.production.min.js:18
    React 4
    unstable_runWithPriority scheduler.production.min.js:18
    React 5
[react-dom.production.min.js:216:199](webpack:///node_modules/react-dom/cjs/react-dom.production.min.js)
```

**Comment by rybaha** [comment]
+1 to this issue. After changing log level - empty log window until page refresh.

**Comment by susnux** [comment]
Fixed on master with #936
